### PR TITLE
Improve JS testing

### PR DESCRIPTION
**Pull request by thomasiles**
This PR adds a linter and test framework to the project.

The project currently doesn't have any javascript beyond the govuk-frontend standard. We will need better testing when adding cookie consent and analytics javascript, which will be in further commits.

Jest and Standard were chosen as they are used across GDS for client-side javascript testing and cover our needs.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18.15.0
```
Comment (lfdebrux) [inline .nvmrc:1]: We might want to update the README, it currently says we use Node 16.

**Review by lfdebrux — COMMENTED** [review]
Looks good to me! Can we get these new test commands added to the README?

File: .nvmrc (hunk 1 above)
Comment (thomasiles) [reply .nvmrc:1]: good point!

**Review by lfdebrux — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -29,13 +34,37 @@ yarn
 
 # Then either:
 # 3a. Run the 'dev' task to start a local server for development
-yarn run dev
+yarn run start
```
[the remark this answers is not in the document]
Comment (lfdebrux) [reply README.md:37]: I was going to raise a PR to fix this, thanks for doing it first 😁

**Pull request merged**